packages/hash/api/src/model/knowledge/link.model.ts

```diff
@@ -39,6 +39,37 @@ export default class {
     this.targetEntityModel = targetEntityModel;
   }
 
+  static async fromPersistedLink(
+    graphApi: GraphApi,
+    { sourceEntityId, targetEntityId, linkTypeUri }: PersistedLink,
+  ): Promise<LinkModel> {
+    const [sourceEntityModel, targetEntityModel, linkTypeModel] =
+      await Promise.all([
+        EntityModel.getLatest(graphApi, { entityId: sourceEntityId }),
+        EntityModel.getLatest(graphApi, { entityId: targetEntityId }),
+        LinkTypeModel.get(graphApi, { versionedUri: linkTypeUri }),
+      ]);
+
+    return new LinkModel({
+      /** @todo: figure out how this can be obtained from `PersistedLink`, or whether it should be removed from the `LinkModel` class */
+      accountId: "",
```

Comment:
@TimDiekmann @Alfred-Mountfield any thoughts what I should do here? Do either of you have insight on why the `LinkModel` class has an `accountId`, and the `PersistedLink` does not?

Reply:
I think it's a bug, and we should return the `createdById`

Reply:
Is there a task for this? I should probably link it in the `@todo`. `accountId: ""` is something we should try to not keep around for long.

Reply:
Weren't you talking to @TimDiekmann today about `PersistedLink` or was that something else?